packages/fast-components-react-base/src/label/label.schema.json

```diff
@@ -7,7 +7,7 @@
     "children": true,
     "properties": {
         "tag": {
-            "title": "HTML tag",
+            "title": "Element",
```

Comment:
HTML Element or just Element? @chrisdholt do you have an opinion?

Reply:
The notes said "Element"

Reply:
I think we need to be specific regarding this being an HTML Element.

Reply:
Changed